src/index.js

```diff
@@ -1970,17 +1974,14 @@ const initializeFormElements = () => {
       }
     } catch (err) {
       console.error(err);
-      signTypedDataV3VerifyResult.innerHTML = `Error: ${err.message}`;
+      signTypedDataVerifyResult.innerHTML = `Error: ${err.message}`;
     }
   };
 
   /**
    * Sign Typed Data Version 3 Test
    */
   signTypedDataV3.onclick = async () => {
-    const networkId = parseInt(networkDiv.innerHTML, 10);
-    const chainId = parseInt(chainIdDiv.innerHTML, 16) || networkId;
-
```

Comment:
after factorizing the code, we can remove all these blocks from all the signatures, as we now have a variable holding the chain id integer